ldk/javascript/src/whisper/react/component-handlers.ts

```diff
@@ -0,0 +1,258 @@
+};
+const radioGroupHandler: ComponentSpecificHandler = {
+  helpsType: WhisperComponentType.RadioGroup,
+  whisperTagType: 'oh-radio-group',
+};
+const richTextEditorHandler: ComponentSpecificHandler = {
+  helpsType: WhisperComponentType.RichTextEditor,
+  whisperTagType: 'oh-rich-text-editor',
+};
+const sectionTitleHandler: ComponentSpecificHandler = {
+  helpsType: WhisperComponentType.SectionTitle,
+  whisperTagType: 'oh-section-title',
+};
+const selectHandler: ComponentSpecificHandler = {
+  helpsType: WhisperComponentType.Select,
+  whisperTagType: 'oh-select',
+};
+const telephoneHandler: ComponentSpecificHandler = {
+  helpsType: WhisperComponentType.Telephone,
+  whisperTagType: 'oh-telephone',
+};
+const textInputHandler: ComponentSpecificHandler = {
+  helpsType: WhisperComponentType.TextInput,
+  whisperTagType: 'oh-text-input',
+};
+const whisperHandler: ComponentSpecificHandler = {
+  appendInitialChild: createAppendFunction('components'),
+  helpsType: 'whisper' as any,
+  whisperTagType: 'oh-whisper',
+};
+
+function assertUnreachable(_: never): never {
+  throw new Error(`Unexpected Type: ${_}`);
+}
+
+// eslint-disable-next-line consistent-return
+export function getHandlerByHelpsType(type: ComponentTypeWithWhisper): ComponentSpecificHandler {
+  switch (type) {
+    case 'whisper':
+      return whisperHandler;
+    case WhisperComponentType.Autocomplete:
+      return autoCompleteHandler;
+    case WhisperComponentType.Box:
+      return boxHandler;
+    case WhisperComponentType.Button:
+      return buttonHandler;
+    case WhisperComponentType.Checkbox:
+      return checkboxHandler;
+    case WhisperComponentType.CollapseBox:
+      return collapseBoxHandler;
+    case WhisperComponentType.DateTimeInput:
+      return dateTimeHandler;
+    case WhisperComponentType.Divider:
+      return dividerHandler;
+    case WhisperComponentType.DropZone:
+      return dropZoneHandler;
+    case WhisperComponentType.Email:
+      return emailHandler;
+    case WhisperComponentType.Icon:
+      return iconHandler;
+    case WhisperComponentType.Link:
+      return linkHandler;
+    case WhisperComponentType.ListPair:
+      return listPairHandler;
+    case WhisperComponentType.Markdown:
+      return markdownHandler;
+    case WhisperComponentType.Message:
+      return messageHandler;
+    case WhisperComponentType.Number:
+      return numberHandler;
+    case WhisperComponentType.Password:
+      return passwordHandler;
+    case WhisperComponentType.RadioGroup:
+      return radioGroupHandler;
+    case WhisperComponentType.RichTextEditor:
+      return richTextEditorHandler;
+    case WhisperComponentType.Select:
+      return selectHandler;
+    case WhisperComponentType.Telephone:
+      return telephoneHandler;
+    case WhisperComponentType.TextInput:
+      return textInputHandler;
+    case WhisperComponentType.SectionTitle:
+      return sectionTitleHandler;
+    default:
+      assertUnreachable(type);
```

Comment:
I understand that it is a last statement, but for the sake if anyone will be extending this functionality, will it make more sense to add `break;` at the end of the default case?